### ISSUE TITLE
The default naming scheme does not play well with JupyterLab

### ISSUE THREAD

**Issue by Cadair** [Cadair/jupyter_environment_kernels#37]
![jupyterlab](https://user-images.githubusercontent.com/1391051/42514013-fe70993e-841d-11e8-9467-0adaf9e9d16e.png)

All I see is "Environment".

@jankatins Do you think it's worth changing the naming scheme to be something like `name of env (Environment)` or maybe `name of env (conda)` or something.

**Comment by jankatins** [comment]
Yes, I actually have it changed on my local system like that. Just never wanted to touch the defaults because others might rely on it.
